cfn.yaml

```diff
@@ -49,10 +51,144 @@ Resources:
               Action:
                 - lambda:InvokeFunction
               Resource: "*"
-  Lambda:
+        - PolicyName: sqs
+          PolicyDocument:
+            Statement:
+              Effect: Allow
+              Action:
+                - sqs:DeleteMessage
+                - sqs:DeleteMessageBatch
+                - sqs:ReceiveMessage
+                - sqs:SendMessage
+                - sqs:SendMessageBatch
+              Resource:
+                - !GetAtt NewsletterCutOffDateQueue.Arn
+                - !GetAtt CleanseListQueue.Arn
```

Comment:
I suppose if we wanted to be totally strict then we'd have separate roles for the lambdas that require access to one queue or the other (for send/receive).